assets/src/domain/eventEditor/ui/ticketAssignmentsManager/components/table/DateCell.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { parseISO } from 'date-fns';
+import classNames from 'classnames';
+
+import { RenderCellProps } from '../../types';
+import { getBackgroundColorClassName } from '@sharedEntities/datetimes/helpers';
+import { useTimeZoneTime } from '@appServices/hooks';
+import { blaBlaFormat } from '@appConstants/dateFnsFormats';
+
+const DateCell: React.FC<RenderCellProps> = ({ datetime }) => {
+	const bgClassName = getBackgroundColorClassName(datetime);
+	const { formatForSite: format } = useTimeZoneTime();
+	const stripeClassName = classNames('date-stripe', bgClassName);
+	const startDate = format(parseISO(datetime.startDate), blaBlaFormat);
+
+	return (
+		<>
+			<div className={stripeClassName}></div>
+			<div className='ee-focus-priority-8'>ID: {datetime.dbId}</div>
+			<div className='ee-focus-priority-6 date-cell__name'>{datetime.name}</div>
+			<div className='ee-focus-priority-5'>{startDate}</div>
```

Comment:
priority 5 would make the date larger than the name

```suggestion
			<div className='ee-focus-priority-8'>{startDate}</div>
```

Reply:
updated

Reply:
I see your suggested 8, but in requirements, it's 6, which one do you want me to apply?

![image](https://user-images.githubusercontent.com/1477453/78679745-49dfa100-78f3-11ea-9689-767456ee3c9b.png)

Reply:
oh now how did i miss that?!?!?! :facepalm:

plz follow those detailed instructions then because that's what I worked about when manipulating the CSS in the DOM

I think what happened then is that you just had the two priorities flipped, likely because I did not have those bullet points in the same order as the elements in the DOM appear (ie: i had: `id`, `startDate`, `name`, whereas the order in the DOM is `id`, `name`, `startDate`) so you had the priorities as 8, 6, 5, but they should be 8, 5, 6. Sorry for the confusing instructions.

Reply:
purfect. it's `8, 5, 6`
![image](https://user-images.githubusercontent.com/1477453/78691990-f294fd00-7901-11ea-923f-4f5b308f6e3c.png)